docker/Dockerfile.distroless

```diff
@@ -1,5 +1,5 @@
 # prepare a distroless source context to copy files from
-FROM gcr.io/distroless/static-debian11@sha256:6706c73aae2afaa8201d63cc3dda48753c09bcd6c300762251065c0f7e602b25 as distroless_source
```

Comment:
may I know the exact change for this version?

Reply:
not sure how to find the exact commit that is resposible for this change, btw closed this one and created a fresh PR. Let's discuss it there.